components/test/src/main/java/org/trellisldp/test/AbstractResourceServiceTests.java

```diff
@@ -24,5 +24,6 @@
      * Get the ResourceService.
      * @return the resource service
      */
+    @Override
     public abstract ResourceService getResourceService();
```

Comment:
Just wondering if this abstract class is needed as it doesn't implemented any methods or have any common code.

Reply:
@jholleran that's a very good point, and the answer is probably that we don't really need that class. What we should do is probably take a broader look at the structure of the test suite. While I think the use of all these JUnit5 `@Nested` classes is very clever, it may not be the best structure for a generalizable test suite. I'll open a separate issue to review the test component.